examples/projections/nongeo/polar.py

```diff
@@ -152,15 +159,13 @@
     # instead of standard angle, rotate coordinate system counterclockwise by
     # 45 degrees, r-axis is marked as depth
     projection="P5c+a+t45+z",
-    # set the frame and color
-    frame=["xa30f", "ya", "WNse+gbisque"],
-)
-
-fig.text(
-    position="TC", text=r"projection='P5c+a\+t45+z'", offset="0/2.0c", no_clip=True
-)
-fig.text(
-    position="TC", text="region=[0, 90, 3480, 6371]", offset="0/1.5c", no_clip=True
+    # set the frame, color, and title
+    # @^ allows for a line break within the title
+    frame=[
+        "xa30f",
+        "ya",
+        "WNse+gbisque+tprojection='P5c+a+t45+\z' @^ region=[0, 90, 3480, 6371]",  # noqa:W605
```

Comment:
This should work and is a little better:
```suggestion
        "WNse+gbisque+tprojection='P5c+a+t45+\\z' @^ region=[0, 90, 3480, 6371]",
```

Reply:
Thanks!